app/model.py

```diff
@@ -636,6 +636,17 @@ def get_unreviewed_notes_count(repo, filter_expired=True):
             ).filter('reviewed =', False).filter('hidden =', False)
         return query.count()
 
+    @staticmethod
+    def put_note(repo, entity):
```

Comment:
Optional: Would be nice to define a similar method for Person as well for consistency. It will probably be called just once, but it will help people to read code if the two similar logic is implemented in a similar method.